src/Synapse/Synapse/Commands/ManagementCommands/SqlPool/StartAzureSynapseVulnerabilityAssessmentScan.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Synapse.Common;
+using Microsoft.Azure.Commands.Synapse.Models;
+using Microsoft.Azure.Commands.Synapse.VulnerabilityAssessment.Model;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Synapse
+{
+    [Cmdlet("Start", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + SynapseConstants.SynapsePrefix + SynapseConstants.SqlPool + SynapseConstants.VulnerabilityAssessmentScanInitialziation, DefaultParameterSetName = StartSqlPoolObjectParameterSet
+, SupportsShouldProcess = true),OutputType(typeof(PSVulnerabilityAssessmentScanRecordModel))]
+    [OutputType(typeof(PSVulnerabilityAssessmentScanRecordModel))]
+    public class StartAzureSynapseSqlPoolVulnerabilityAssessmentScanInitialziation : SynapseManagementCmdletBase
```

Comment:
>StartAzureSynapseSqlPoolVulnerabilityAssessmentScanInitialziation [](start = 17, length = 65)

Start-AzSynapseSqlPoolVulnerabilityAssessmentScan? Why do you add an Initialziation suffix?

Reply:
ping on this..

---
In reply to: [543111306](https://github.com/Azure/azure-powershell/pull/13690#discussion_r543111306) [](ancestors = 543111306)